lib/preview/routes/module.js

```diff
@@ -0,0 +1,73 @@
+const http = require('../../../http')
+const { fetchModulesByPath } = require('../../../api/designManager');
+const {
+  addRefreshScript,
+  getPreviewUrl,
+  memoize,
+  trackPreviewEvent,
+  buildHTMLResponse
+} = require('../previewUtils');
+const { logger } = require('./../../../logger');
+
+const cachedFetchModulesByPath = memoize(fetchModulesByPath);
```

Comment:
Am I correct in thinking that this code will break if someone deletes a template/module while the proxy server is running? We cache the template, so we assume it will always exist. That may be a difficult bug to track down.

Reply:
Oops yeah - you can see I started writing some cache busting logic for that in `memoize` but something else pulled my attention, I'll just take out the cache for now it's fine without it in the meantime